packages/host/app/services/module-info-service.ts

```diff
@@ -0,0 +1,33 @@
+import Service, { service } from '@ember/service';
+
+import { SupportedMimeType } from '@cardstack/runtime-common';
+
+import LoaderService from '@cardstack/host/services/loader-service';
+
+export default class ModuleInfoService extends Service {
```

Comment:
I don't understand why you need this service--isn't this what realm info service is doing? why can't you just move the cache into that service?

Reply:
is this to cache 302 redirects? seems like a pre-mature optimization to me....